### PR TITLE
Initial set-up for ReadtheDocs documentation for Perseo Front End.

### PR DESCRIPTION
**Pull request by jason-fox**
- Please create a project on readthedocs called fiware-perseo-fe and point to this repository.
- Includes standard CSS and badges.
- Just a dump of existing documentation but with CSS styling.

### CODE REVIEW

File: documentation/index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Perseo-FE (EPL server)
```
Comment (fgalan) [inline documentation/index.md]: "(EPL Server)" should be removed

File: mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+site_name: Perseo Front End
+site_url: https://fiware-perseo-fe.readthedocs.org
+repo_url: https://github.com/telefonicaid/perseo-fe.git
+site_description: Perseo-Front End (EPL server)
```
Comment (fgalan) [inline mkdocs.yml]: "(EPL Server)" should be removed

**Comment by fgalan** [comment]
fiware-perseo-fe project has been created at RTD, although I haven't configured yet automatic integration:

imagen

Do you have at hand the howto about this (I'm not fully sure but I remember there were some piece of documentation in FIWARE docs about it...), pls?

**Comment by rafitanba** [comment]
Hi Jason, 

We (Ficodes) created long time ago https://perseo.readthedocs.io for publishing Perseo documentation. We are working on updating existing documentation to reflect recent changes of Perseo, and have them published there.

Currently we are working in our own repo (https://github.com/Ficodes/perseo-fe/tree/newdocumentation) to have it all ready for next week FIWARE Summit, but then we will create a PR to telefonica’s repo. Transfering the ownership of this readthedocs page to Telefónica is pending yet.

**Comment by fgalan** [comment]
Thanks @rafitanba for the feedback.

I'll hold on the merge of this PR while this gets clarified. I can remove the just created fiware-perseo-fe and fiware-core proyects at RTD if they are inapropiated.

**Comment by rafitanba** [comment]
Hi @fgalan.

Let's wait for @jason-fox thoughts on this matter, but whatever his answer is, I would have just one Perseo project at RTD including both fe and core documentation.

**Comment by fgalan** [comment]
Currently, most of the documentation is location in perseo-fe (and it makes sense). As far as I remember, Perseo Core only includes information about building the component and the minium FE-Core API.

Is possible to configure the same RTD project web-hooked to two different Github repositories? If not possible, having two separarte RTD project (each one associated to a repository) would be feasible.

File: documentation/index.md (hunk 1 above)
Comment (jason-fox) [reply documentation/index.md]: Fixed 6c4b6af

File: mkdocs.yml (hunk 2 above)
Comment (jason-fox) [reply mkdocs.yml]: Fixed 5d12de7

**Comment by jason-fox** [comment]
> Is possible to configure the same RTD project web-hooked to two different Github repositories? If not possible, having two separate RTD project (each one associated to a repository) would be feasible.

I don't think it is, the correct way to do this would be to **either** have the the components submodules and the docs as a root module **or** place the docs in one of the two repos and just place a simple link to the published RTD in the other one.

I've made the minor mods requested in case you want to merge this. Alternatively take the documentation directly from the FI-Codes docs and add close this PR.

**Comment by fgalan** [comment]
Ficodes (@rafitanba @cblanco @aarranz), what do you think? It would be great if we could progress on the final solution for Perseo documentation in RTD...

**Comment by fgalan** [comment]
A new PR (https://github.com/telefonicaid/perseo-fe/pull/330) has been created related with Perseo FE documentation. It overpasses this one? Should this one be closed?

**Comment by jason-fox** [comment]
Probably. Let's keep the branch for reference for now and I'll close it.

**Pull request closed without merging**